google/cloud/spanner_admin_database_v1/proto/spanner_database_admin_pb2.py

```diff
@@ -1489,10 +1489,10 @@
           Number of databases to be returned in the response. If 0 or
           less, defaults to the server’s maximum allowed page size.
       page_token:
-          If non-empty, ``page_token`` should contain a [next\_page\_tok
-          en][google.spanner.admin.database.v1.ListDatabasesResponse.nex
-          t\_page\_token] from a previous [ListDatabasesResponse][google
-          .spanner.admin.database.v1.ListDatabasesResponse].
+          If non-empty, ``page_token`` should contain a [next_page_token
+          ][google.spanner.admin.database.v1.ListDatabasesResponse.next_
+          page_token] from a previous [ListDatabasesResponse][google.spa
```

Comment:
This line wrap [breaks Sphinx](https://source.cloud.google.com/results/invocations/4e95fdae-7c19-4cbe-9a15-ede4c51cfbc5/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-spanner%2Fpresubmit%2Fpresubmit/log):

```python
sphinx.errors.SphinxWarning: /tmpfs/src/github/python-spanner/google/cloud/spanner_admin_database_v1/types.py:docstring of google.cloud.spanner_admin_database_v1.types.ListDatabasesRequest:17:Unknown target name: "google.spanner.admin.database.v1.listdatabasesresponse.next".
```

Reply:
I don't think the issue is the line wrap. I think it is the change to not escaping the underscores. I filed an issue for it:
https://github.com/googleapis/synthtool/issues/660